README.md

```diff
@@ -720,23 +720,30 @@ interface Table extends Parent {
 ### CustomCodeComponent
 
 ```ts
-interface CustomCodeComponent extends Parent {
+interface CccFallbackText extends Node {
+    type: 'ccc-fallback-text'
+    children: Paragraph[]
+}
+
+type CccAttributes = {
```

Comment:
Thought: Same as before, why this need a CCC prefix given the type that represents?

Reply:
question: why is it a separate type and not `children: ImageSet[] | Paragraph[]`